packages/xo-web/src/xo-app/settings/servers/index.js

```diff
@@ -100,17 +100,16 @@ const COLUMNS = [
     itemRenderer: server => (
       <div>
         <StateButton
-          disabledLabel={_('serverDisconnected')}
+          disabledLabel={_('serverDisabled')}
           disabledHandler={connectServer}
```

Comment:
This is no longer what this button should do, it should change the `enabled` state of the server.

Update the API if needed.